src/settings/OpeningPeriodForm/OpeningPeriodFormWrapper.js

```diff
@@ -123,13 +123,18 @@ class OpeningPeriodFormWrapper extends Component {
     });
   }
 
-  catchOverlappedEvents = async err => {
-    if (err.status === 422) {
+  catchOverlappedEvents = async (err) => {
+    if (err.status >= 400) {
```

Comment:
In bug we have description only for one case for `Overlapping error`. How i understand if we want localize all `errors.code` we should describe this in the bug, get all `errors.code` from BE team and provide translation for them. Could you please double check should we change from `err.status === 422`  to `err.status >= 400` in scope of current pull request?

Reply:
I figured that a more generic error check would be preferable to one that handled only this status code, as, later on, if BE added other error codes, we would be able to show messages for them without any additional changes to this code.

Reply:
@Dmitriy-Litvinenko also, the only error message that the BE is configured to send is `intervalsOverlap`: https://github.com/folio-org/mod-calendar/blob/df29359584b5d52b2bcc516ef3568085e6b4c4d0/src/main/java/org/folio/rest/utils/CalendarUtils.java#L369 .